pkg/kubectl/cmd/get.go

```diff
@@ -278,8 +278,12 @@ func RunGet(f cmdutil.Factory, out, errOut io.Writer, cmd *cobra.Command, args [
 				objsToPrint = append(objsToPrint, obj)
 			}
 			for _, objToPrint := range objsToPrint {
-				if err := printer.PrintObj(objToPrint, writer); err != nil {
-					return fmt.Errorf("unable to output the provided object: %v", err)
+				if isFiltered, err := filterFuncs.Filter(objToPrint, filterOpts); !isFiltered {
+					if err != nil {
+						glog.V(2).Infof("Unable to filter resource: %v", err)
```

Comment:
Why not just return an error here or use glog.Errorf?

Reply:
Took this from the existing filter [here](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/get.go#L407), but can switch to `glog.Errorf` if that works better. Returning an error will terminate early, potentially not printing any remaining resources (or a filter error for each of them)

Reply:
As long as there's precedence, that works for me.

Reply:
We don't use `glog` in the CLI for anything other that leveled logging (`glog.V(n)...`) .